pkg/router/mine-service.go

```diff
@@ -15,6 +15,7 @@ func MineServiceUpload(r *gin.Engine, validate *validator.Validate, ApiVersion s
 	authUrl := r.Group(fmt.Sprintf("/api/%v", ApiVersion))
 	{
 		authUrl.POST("/mine-service/upload", mineservice.MineImageUpload)
+		authUrl.POST("/mine-service/upload/prompt", mineservice.PromptMineImageUpload)
```

Comment:
upload-prompt is better than upload/prompt
upload is not a path, it is the action